connectors/proofpoint/configurations/lang_en.json

```diff
@@ -17,14 +17,14 @@
                 "label": "Time (UTC)",
                 "description": "Set the UTC time to run connector to pull data from data source"
             }
+        },
+        "host": {
+            "label": "ProofPoint_URL",
```

Comment:
This may have been missed during the initial review, but this label should be properly formatted since it's user-facing in the UI. It doesn't need the "Proofpoint" name since we are trying to keep these things consistent. If you want to have it the same as in UDI, it would be `Management IP address or Hostname`.